src/main/kotlin/org/rust/ide/annotator/RsEdition2018KeywordsAnnotator.kt

```diff
@@ -8,13 +8,15 @@ package org.rust.ide.annotator
 import com.intellij.lang.annotation.AnnotationHolder
 import com.intellij.psi.PsiElement
 import org.rust.ide.colors.RsColor
+import org.rust.ide.utils.isEnabledByCfg
 import org.rust.lang.core.psi.*
 import org.rust.lang.core.psi.RsElementTypes.IDENTIFIER
 import org.rust.lang.core.psi.ext.*
 
 class RsEdition2018KeywordsAnnotator : RsAnnotatorBase() {
     override fun annotateInternal(element: PsiElement, holder: AnnotationHolder) {
         if (!isEdition2018Keyword(element)) return
+        if (!element.isEnabledByCfg) return
```

Comment:
Maybe we want to do it more accurately?
It's quite inconsistent that all keywords are bold even under cfg except `try` and `async`

Reply:
Done https://github.com/intellij-rust/intellij-rust/pull/4575